src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/Helpers.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+using IServiceProvider = Microsoft.VisualStudio.OLE.Interop.IServiceProvider;
+
+namespace Microsoft.VisualStudio.RazorExtension.SyntaxVisualizer
+{
+    internal static class Helpers
```

Comment:
`Helpers` is not a super informative name if it shows up in your Intellisense. Maybe `VSServiceHelpers`?